mula/scheduler/models/scheduler.py

```diff
@@ -8,5 +8,5 @@ class Scheduler(BaseModel):
     unmarshalling of schedulers to a JSON representation."""
 
     id: Optional[str]
-    populate_queue_enabled: Optional[bool]
+    enabled: Optional[bool]
```

Comment:
I was wondering why this property is optional. What's the expected behavior if it's unset?

Reply:
This optional is mainly used for the serialization of the model when patching. Might need to think about what models are used as a database representation and what are used for REST response models, and have a clearer definition and separation of those.

Reply:
If you're using this entire structure/ entity in the `PATCH` request, then you might want to consider having a default value instead.

Reply:
No not the entire structure. Think the only viable field for patching is `enabled` at the moment

Reply:
Yes, but the current endpoint also allows `{"enabled": null}" which doesn't do anything which is unusual for a `PATCH` request.

Different representations are indeed a good idea. It's also common to have different request and response models. I'll leave it up to you for now.

Reply:
Ah I see. I'll have to look into it if this relates to the omission of the field (key/value) in a patch request or the omission of the value. But I'll up the suggestion of the different representations in another issue, which can be a nice addition to make the differentiation more clear.

Reply:
> Yes, but the current endpoint also allows
> 
> ```json
> {"enabled": null}
> ```
> 
> which doesn't do anything which is unusual for a `PATCH` request.
> 
> Different representations are indeed a good idea. It's also common to have different request and response models. I'll leave it up to you for now.

that json request will not be allowed by the current endpoint.

```
{
    "detail": [
        {
            "loc": [
                "body",
                "enabled"
            ],
            "msg": "value could not be parsed to a boolean",
            "type": "type_error.bool"
        }
    ]
}
```